vip-helpers/vip-deprecated.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/**
+ * Loads the built-in WP REST API endpoints in WordPress.com VIP context.
+ *
+ * @deprecated Not applicable since VIP 2.0.0
+ */
+function wpcom_vip_load_wp_rest_api() {
```

Comment:
Can we move this to https://github.com/Automattic/vip-go-wpcom-compat instead? And make sure all migrating sites load that plugin?

Reply:
Yep!  I'll close this out and make a PR there.  Thanks :)